modules/foodRandomSelection.js

```diff
@@ -38,13 +32,20 @@ const foodRandomSelection = {
           </div>
           <p class="likes">likes</p>
           <div class="buttons">
-            <input type="button" class="button" value="Comments">
-            <input type="button" class="button" value="Reservations">
+            <input type="button" class="button0" value="Comments">
+            <input type="button" class="button1" value="Reservations">
           </div>
         </div>
       `).join('');
     });
   },
+
 };
 
-export default foodRandomSelection;
\ No newline at end of file
+const exporting = () => { // eslint-disable-line
+  mealsArray; // eslint-disable-line
+}; // eslint-disable-line
```

Comment:
Brilliant idea to create a variable. But to understand the code can you please edit it as follows:
**// eslint-disable-next-line space-infix-ops**
const exporting = () => { 
   **// eslint-disable-next-line space-infix-ops**
  mealsArray; 
};